custom_components/tuya_local/devices/tv_backlights.yaml

```diff
@@ -0,0 +1,223 @@
+name: TV Sync Backlights
+products:
+  - id: gaobbrxqiblcng2p
+    name: Loginovo Ambient TV Sync Light
+primary_entity:
+  entity: light
+  name: Backlight
+  icon: mdi:television-shimmer
+  dps:
+    - id: 102
+      type: boolean
+      name: switch
+    - id: 104
+      name: brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+          constraint: video_mode
+          conditions:
+            - dps_val: ambi_mode
+              value_redirect: scene_brightness
+            - dps_val: color_mode
+              value_redirect: color_brightness
+    - id: 114
+      hidden: true
+      name: scene_brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+    - id: 112
+      hidden: true
+      name: color_brightness
+      type: integer
+      range:
+        min: 0
+        max: 100
+      mapping:
+        - step: 1
+          scale: 0.392
+    - id: 111
+      name: rgbhsv
+      type: hex
+      format:
+        - name: r
+          bytes: 1
+        - name: g
+          bytes: 1
+        - name: b
+          bytes: 1
+    - id: 101
+      name: video_mode
+      type: string
+      hidden: true
+    - id: 101
+      name: effect
+      type: string
+      mapping:
+        - dps_val: video_mode
+          value: Video
+        - dps_val: music_mode
+          value: Music
+        - dps_val: film_mode
+          value: Wide film
+        - dps_val: color_mode
+          value: Static color
+        - dps_val: ambi_mode
+          value: Ambient scene
+secondary_entities:
+  - entity: number
+    name: Backlight intensity
+    type: integer
+    mode: slider
+    icon: mdi:speedometer-medium
+    dps:
+      - id: 103
+        name: value
+        type: integer
+        range:
+          min: 1
+          max: 3
+        mapping:
+          - step: 1
+            scale: 1
+  - entity: select
+    icon: mdi:shimmer
+    name: Scene select
+    format:
```

Comment:
This does not belong here.  It needs to be specified on a specific dp (if at all)